core/lib/spree/core/controller_helpers/store.rb

```diff
@@ -18,6 +18,20 @@ def current_store
           @current_store ||= Spree::Store.current(request.env['SERVER_NAME'])
         end
 
+        # Return a Hash of things that influence the prices displayed in your shop.
+        #
+        # By default, the only thing that influences prices that is the current order's +tax_zone+
+        # (to facilitate differing prices depending on VAT rate for digital products in Europe, see
+        # https://github.com/spree/spree/pull/6295 and https://github.com/spree/spree/pull/6662).
+        #
```

Comment:
Could you please also include this last bit from the release notes in the comments here as well:
```
If your prices depend on something else, overwrite this method and add more key/value pairs to the Hash it returns.

Be careful though to also patch the following parts of Spree accordingly:

* `Spree::VatPriceCalculation#gross_amount`
* `Spree::LineItem#update_price`
* `Spree::Stock::Estimator#taxation_options_for`
* Subclass the `DefaultTax` calculator
```